crates/net/network/src/test_utils/testnet.rs

```diff
@@ -291,7 +291,7 @@ impl<C, Pool> TestnetHandle<C, Pool> {
     /// Connects all peers with each other
     pub async fn connect_peers(&self) {
         if self.peers.len() < 2 {
-            return
+            return;
```

Comment:
are you running rustfmt with nightly toolchain? `cargo +nightly fmt`. make sure rust is up to date to `rustup update`.